contracts/mocks/DCAHubCompanion/DCAHubCompanionHubProxyHandler.sol

```diff
@@ -3,4 +3,49 @@ pragma solidity >=0.8.7 <0.9.0;
 
 import '../../DCAHubCompanion/DCAHubCompanionHubProxyHandler.sol';
 
-contract DCAHubCompanionHubProxyHandlerMock is DCAHubCompanionHubProxyHandler {}
+contract DCAHubCompanionHubProxyHandlerMock is DCAHubCompanionHubProxyHandler {
```

Comment:
Tried it with smock, but it didn't detect the internal call (even though the function is public) 😢